src/main/java/com/nha/abdm/wrapper/facade/controller/HIP.java

```diff
@@ -2,33 +2,52 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.nha.abdm.wrapper.hrp.CommonHelpers.ResponseHelper;
+import com.nha.abdm.wrapper.hrp.CommonHelpers.VerifyOtp;
+import com.nha.abdm.wrapper.hrp.hipInitiatedLinking.responses.AddPatient;
+import com.nha.abdm.wrapper.hrp.hipInitiatedLinking.responses.LinkRecordsResponse;
 import com.nha.abdm.wrapper.hrp.manager.WorkflowManager;
-import java.io.FileNotFoundException;
-import java.net.URISyntaxException;
-import com.nha.abdm.wrapper.hrp.serviceImpl.PatientTableService;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
+import java.io.FileNotFoundException;
+import java.net.URISyntaxException;
+import java.text.ParseException;
+import java.util.Objects;
 
 @RestController
 public class HIP {
```

Comment:
As a best practice, the names of the controller class should be suffixed with `Controller` keyword, i.e. here the name could be `HIPController`.

Reply:
Modified the Class name to HIPController